<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>io_config_dialog</class>
 <widget class="QDialog" name="io_config_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>447</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog.InputOverlay.Title</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_features">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>6</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>134</height>
      </size>
     </property>
     <property name="title">
      <string>Dialog.InputOverlay.LocalFeatures</string>
     </property>
     <widget class="QCheckBox" name="cb_enable_history">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dialog.InputOverlay.EnableInputHistory</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_local_features">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dialog.InputOverlay.LocalFeatures.Label</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_gamepad_hook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>225</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dialog.InputOverlay.EnableGamepadHook</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_iohook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>194</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dialog.InputOverlay.EnableUiohook</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_enable_overlay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>220</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dialog.InputOverlay.EnableInputOverlay</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_remote">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Dialog.InputOverlay.RemoteConnection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cb_enable_remote">
        <property name="text">
         <string>Dialog.InputOverlay.EnableRemoteConnection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_log">
        <property name="text">
         <string>Dialog.InputOverlay.RemoteConnection.Logging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_port">
        <property name="text">
         <string>Dialog.InputOverlay.RemoteConnection.Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="box_port">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1608</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_status">
        <property name="text">
         <string>Dialog.InputOverlay.RemoteConnection.Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_connections">
        <property name="text">
         <string>Dialog.InputOverlay.RemoteConnection.Connections</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="box_connections">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>io_config_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>io_config_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
